web_responsive/static/src/css/web_responsive.scss

```diff
@@ -324,19 +344,67 @@ html .o_web_client .o_main .o_main_content {
 .o_content, .modal-content {
     max-width: 100%;
 
+    // Sticky Header & Footer in List View
+    >div {
+        >.table-responsive {
+            overflow-x: initial;
+
+            >.o_list_view {
+                // th & td are here for compatibility with chrome
+                thead, thead tr:nth-child(1) th {
```

Comment:
So if a `thead` has multiple rows, why only the 1st one is sticky? :thinking: 

Also, if `thead th` works for other browsers than Chrome too, then can we just leave that?

(the same questions apply to other similar selectors below)